content/kots-cli/get/config.md

```diff
@@ -15,13 +15,13 @@ kubectl kots get config [flags]
 
 - _Provide `[flags]` according to the table below_
 
-| Flag              | Type   | Description                                                         |
-| :---------------- | ------ | ------------------------------------------------------------------- |
-| `--appslug`       | string | The app slug used to retrieve the config for _(required)_                        |
-| `--sequence`      | int    | The app sequence used to retrieve the config for _(required)_                    |
-| `--decrypt`       | bool   | Decrypt password items within the config                            |
-| `-h, --help`      |        | help for get apps                                                   |
-| `-n, --namespace` | string | the namespace where the admin console is running _(required)_       |
+| Flag              | Type   | Description                                                                                       |
+| :---------------- | ------ |---------------------------------------------------------------------------------------------------|
+| `--appslug`       | string | The app slug for which the config will be retrieved _(required if more than one app is deployed)_ |
+| `--sequence`      | int    | The app sequence for which the config will be retrieved _(default: latest)_                       |
+| `--decrypt`       | bool   | Decrypt password items within the config                                                          |
+| `-h, --help`      |        | help for get config                                                                               |
+| `-n, --namespace` | string | the namespace where the admin console is running _(required)_                                     |
```

Comment:
We don't seem to have consistency across this command (or others) about whether the descriptions should start with a capital letter or not. I imagine they should, but I'll let the docs team decide. Even if we update here, other commands appear inconsistent too.

Reply:
Great catch--and it looks like punctuation isn't consistent either. Added a commit to edit both